### PR TITLE
Add Change of Name submission test

### PR DESCRIPTION
**Pull request by danfalc**

### CODE REVIEW

File: cypress/support/page_objects/ChangeOfCompanyName.js [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import BasePage from "./generic/BasePage";
+
+class ChangeOfCompanyName extends BasePage {
+
+    enterProposedName(name) {
+        cy.get('input[type=text]').type(name);
```
Comment (rsuller) [inline cypress/support/page_objects/ChangeOfCompanyName.js]: Does `text` need to be wrapped in quotes here?
Also is `enterProposedName` and `enterResolutionDate` essentially the same? Do we have a case for wrapping up these as a single method either in this class or even elsewhere if similar is used throughout this project?

Comment (danfalc) [reply cypress/support/page_objects/ChangeOfCompanyName.js]: It works fine without the quotes.

Comment (danfalc) [reply cypress/support/page_objects/ChangeOfCompanyName.js]: Thought I added in an additional comment here, but anyway, the two methods are for 2 fields on different tabs of this screen. So I don't thing a single method would be appropriate?

Comment (rsuller) [reply cypress/support/page_objects/ChangeOfCompanyName.js]: OK, let's go with it for now

**Pull request merged**